src/widget/stock_summary.rs

```diff
@@ -95,37 +102,37 @@ impl CachableWidget<StockState> for StockSummaryWidget {
                         },
                         Style::default()
                             .add_modifier(Modifier::BOLD)
-                            .fg(Color::Yellow),
+                            .fg(THEME.text_primary),
                     ),
                 ]),
                 Spans::from(vec![
-                    Span::styled("h: ", Style::default()),
+                    Span::styled("h: ", Style::default().fg(THEME.text_normal)),
                     Span::styled(
                         if loaded {
                             format!("{:.2}", high)
                         } else {
                             "".to_string()
                         },
-                        Style::default().fg(Color::LightCyan),
```

Comment:
Given the defaults for `THEME.text_secondary`, this is no longer `LightCyan`

Reply:
for historical reference, here's the diff

| ![Screenshot_20210217_023921](https://user-images.githubusercontent.com/16881812/108144152-5e6ef700-70c9-11eb-807c-03aa1b990f07.png) | ![Screenshot_20210217_023941](https://user-images.githubusercontent.com/16881812/108144171-6af34f80-70c9-11eb-97ca-9858458d79e5.png) |
| - | - |
| `0.11.0` | [`#69`](https://github.com/tarkah/tickrs/pull/69) |